### PR TITLE
add pip files and define binaries

### PR DESCRIPTION
**Pull request by azazellochg**
I will wait till pip files format / doc is stabilized by Madrid team..

### CODE REVIEW

**Comment by delarosatrevin** [comment]
It seems fine to me. I like your idea to wait for the structure/definition is stable and we can go over all the plugins we maintain. ;)

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    # https://packaging.python.org/specifications/core-metadata/#home-page-optional
+    url='https://github.com/scipion-em/scipion-em-gctf',  # Optional
+
+    # This should be your name or the name of the organization which owns the
+    # project.
+    author='Grigory Sharov',  # Optional
+
+    # This should be a valid email address corresponding to the author listed
+    # above.
+    author_email='scipion@cnb.csic.es',  # Optional
```
Comment (pconesa) [inline setup.py]: Aren't you the author?

File: CHANGES.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+1.0.0 - First version of the plugin
```
Comment (delarosatrevin) [inline CHANGES.txt]: Maybe we keep plugins version 0.0.x until we have a first official release of the pluginization and we bump it to 1.0.0?

File: setup.py (hunk 1 above)
Comment (delarosatrevin) [reply setup.py]: I think so.... ;)

File: CHANGES.txt (hunk 2 above)
Comment (pconesa) [reply CHANGES.txt]: I think first plugins stable version make sense to be 1.0.0--so we should start with less. ANd, yes bump it when happy with it....we have soon to modify buildbot to understand new structure, test each plugin and report back to, ideally and specific slack channel.

**Pull request merged**